src/Question.js

```diff
@@ -76,6 +82,7 @@ class Question extends React.Component {
                             <Form.Label>
                                 Question
                             </Form.Label>
+                            {this.props.length > 1 && <button className='danger' onClick={this.deleteQuestion}>Delete Question</button>}
```

Comment:
This is super nitpicky, but it's weird for `<Question />` to know about the parent length. I'd suggest modeling this as a `canDelete` prop, with the _parent_ component deciding if delete is allowed, i.e. `canDelete={questions.length > 1}`.